modules/metabat2/metabat2/main.nf

```diff
@@ -33,8 +36,19 @@ process METABAT2_METABAT2 {
 
     mv metabat2/${prefix} ${prefix}.tsv
     mv metabat2 bins
-    bgzip --threads $task.cpus ${prefix}.tsv
-    bgzip --threads $task.cpus bins/*.fa
+
+    gzip ${prefix}.tsv
+    gzip bins/*.fa
```

Comment:
Did some xargs diggging (the container version of xargs is old).
This should work
```suggestion
    find . -type f -name "bins/*.fa" | xargs -t -n 1 bgzip -@ 4
```

Reply:
Will try on my mag test